src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -2,11 +2,15 @@
 
 Release History
 ===============
+
+2.0.21
+++++++
+* fix `aks get-credentials -f` without fully-qualified path
+
 2.0.20
 ++++++
 * `acs create`: emit out an actionable error if provisioning application failed for lack of permissions
 
```

Comment:
Put your new history under 2.0.20 and pre-check should pass.
Master is still on 2.0.19